cps-global-lib/src/main/resources/org/jenkinsci/plugins/workflow/cps/global/UserDefinedGlobalVariable/help.jelly

```diff
@@ -0,0 +1,4 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core">
+  <j:out value="${it.helpHtml}"/>
```

Comment:
Careful—this is user content emitted raw. It would be safer to use `MarkupFormatter`, or restrict to plain text. Admittedly it currently requires `RUN_SCRIPTS` to push to this repo, but see [JENKINS-26538](https://issues.jenkins-ci.org/browse/JENKINS-26538).